doc/source/whatsnew/v1.0.0.rst

```diff
@@ -195,6 +195,7 @@ Documentation Improvements
 
 - Added new section on :ref:`scale` (:issue:`28315`).
 - Added sub-section Query MultiIndex in IO tools user guide (:issue:`28791`)
+- Added hints how setup a dev environment under Windows when using pip only (:issue:`29112`)
```

Comment:
This is a nice idea but OK to remove - we don't add whatsnew entries for documentation-only updates

Reply:
Alright, I was not sure whether or not should add it, will remove it. :)